apps/src/templates/sectionsRefresh/coteacherSettings/RemoveCoteacherDialog.jsx

```diff
@@ -28,15 +27,16 @@ export default function RemoveCoteacherDialog({
         closeRemoveDialog();
         return;
       }
-      $.ajax({
-        url: `/api/v1/section_instructors/${coteacher.id}`,
-        method: 'DELETE',
-      })
-        .done(() => {
+      fetch(`/api/v1/section_instructors/${coteacher.id}`, {
+        type: 'DELETE',
+      }).then(response => {
+        if (response.ok) {
```

Comment:
Don't repeat yourself. :-)

```suggestion
        if (response.ok) {
          removeSavedCoteacher(coteacher.id);
        }
        closeRemoveDialog();
```